### ISSUE TITLE
Template folder is empty

### ISSUE THREAD

**Issue by tyeeman** [wirekang/blogen#2]
Thanks for that last fix. Now when I run blogen -i, these are the file sizes in the template folder - 

<img width="372" alt="2021-10-26_222603" src="https://user-images.githubusercontent.com/4469306/139004991-630f092a-02fe-480b-a176-e3688099d574.png">

**Comment by wirekang** [comment]
Sorry for low quality, It is intended behavior. Check out /example directory.

**Comment by tyeeman** [comment]
No, don't worry about the low quality. My point is that when I run the init function now (blogen -i), blogen does not copy the template files properly. See above image with the size of the files! 7 bytes, 1 byte, 1 byte, 0 bytes! Please run "blogen -i" on your computer and see!

Here are the example template file sizes - much bigger!

<img width="373" alt="2021-10-27_161544" src="https://user-images.githubusercontent.com/4469306/139160805-c98764d0-2ccc-4fdb-be85-2318c56a010a.png">

**Comment by wirekang** [comment]
There no logic like copying template files. It just creates empty files.

**Comment by tyeeman** [comment]
Oh, OK, thanks for the info.
